be/src/util/simd/vstring_function.h

```diff
@@ -202,6 +202,17 @@ class VStringFunctions {
         return {p - begin, i};
     }
 
+    static inline size_t iterate_utf8_with_start_limit(const char* begin, const char* end,
```

Comment:
`iterate_utf8_with_start_limit ` not need, we can call iterate_utf8_with_limit_length and get first